extensions/azure-extensions/src/main/java/io/druid/storage/azure/AzureAccountConfig.java

```diff
@@ -27,23 +27,23 @@
 public class AzureAccountConfig
 {
   @JsonProperty
-  private String protocol = "https";
+  public String protocol = "https";
```

Comment:
they are actually public everywhere and throughout the Druid codebase for configurations such as this one